pkg/devfile/adapters/kubernetes/utils/utils.go

```diff
@@ -16,9 +14,6 @@ import (
 const (
 	// _envProjectsRoot is the env defined for project mount in a component container when component's mountSources=true
 	_envProjectsRoot = "PROJECTS_ROOT"
-
-	// _envDebugPort is the env defined in the runtime component container which holds the debug port for remote debugging
-	_envDebugPort = "DEBUG_PORT"
```

Comment:
Wouldn't it make more sense to handle #6056 in a separate PR (with a more specific test case, which I did not see here)? I don't really see the link between `env.yaml` and the Debug port env var here ;-)

Reply:
Ah, I see: the debug port used to be stored in `env.yaml`, right? In this case, I think what's missing (unless I missed it) is a test case for https://github.com/redhat-developer/odo/issues/6056 to ensure `odo dev` does not inject a `DEBUG_PORT` env var into the container.

Reply:
Yes, this is the reason why it is done here. Let me check if a test exists for this case